### DISCUSSION TITLE
thinkphp5.1

### DISCUSSION THREAD

**Discussion by yorkgometer** [yansongda/pay#726] [Q&A]
麻烦请问文档中// $container = 您现有的 container中的"您现有的 container"是什么容器？

thinkphp5.1报错如下：
Return value of Yansongda\Pay\Pay::getContainer() must be an instance of Psr\Container\ContainerInterface, instance of think\App returned

**Comment by yansongda** [comment]
按文档安装先：https://pay.yansongda.cn/docs/v3/quick-start/install.html
